scripts/install-operator-results-api.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+set -e
+
+# This script assumes that you are connected to an OpenShift cluster and have the Pipelines Operator installed
+
+echo "Installing the Pipelines Operator ... "
```

Comment:
To give an example, this is how, on ecosystem, we install the operator : https://github.com/openshift-pipelines/task-containers/blob/main/hack/install-osp.sh